qemu/tests/cfg/balloon_check.cfg

```diff
@@ -17,6 +17,16 @@
             no RHEL.3 RHEL.4 RHEL.5 RHEL.6
             no RHEL.7.3 RHEL.7.2 RHEL.7.1 RHEL.7.0
             balloon_opt_deflate_on_oom = yes
+        - balloon_free:
+            only balloon_evict_and_enlarge
+            required_qemu = [5.2.0, )
+            no RHEL.8.3 RHEL.8.2 RHEL.8.1 RHEL.8.0 RHEL.7
+            balloon_opt_free_page_reporting = yes
+            get_res_cmd = "top -n1 -b -p %s | grep qemu-kvm | awk -F ' ' '{print $6}'"
+            consumed_mem = 2G
+            ppc64,ppc64le:
+                onsumed_mem = 4G
```

Comment:
Typo, `consumed_mem`

Reply:
Oh, it must be some paste error. Updated, thanks.